core/common/src/main/java/alluxio/Constants.java

```diff
@@ -73,7 +73,7 @@ public final class Constants {
   public static final String HEADER_HTTPS = "https://";
   public static final String HEADER_KODO = "kodo://";
   public static final String HEADER_OSS = "oss://";
-  public static final String HEADER_OZONE = "o3fs://";
+  public static final String HEADER_OZONE = "o3fs://,ofs://";
```

Comment:
Would you please make this configurable? Since `ofs` is a scheme which conflict with another ufs in our scenario.

Reply:
done